src/modules/discovery/pages/leaderboard/Leaderboard.tsx

```diff
@@ -104,7 +104,6 @@ const MobileLeaderboard = ({ period }: { period: LeaderboardPeriod }) => {
         id="leaderboard-mobile"
         wrapperId="leaderboard-mobile-wrapper"
         offset={dimensions.projectNavBar.mobile.height + 20}
-        bias={-70}
```

Comment:
When you scroll fast, it goes past the normal position and then comes back.

I recommend adding a positive bias of 40 or 50, to solve the flickering to some extent.